main.tf

```diff
@@ -1,40 +1,60 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+data "aws_secretsmanager_secret" "db_secret" {
+  arn = "arn:aws:secretsmanager:us-west-2:659808933566:secret:db-secret-J1Z9s0"
+}
+
+data "aws_secretsmanager_secret_version" "db_secret_version" {
+  secret_id = data.aws_secretsmanager_secret.db_secret.id
+}
+
+
 module "networking" {
   source = "./modules/networking"
 
-  app-name                = local.app-name
-  id_of_vpc               = module.networking.id_of_vpc
-  both_private_subnets_id = [module.networking.both_private_subnets_id]
+  app_name               = local.app_name
+  availability_zones     = slice(data.aws_availability_zones.available.names, 0, 3)
+  cidr                   = "10.0.0.0/16"
+  public_subnets_list    = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+  private_subnets_list   = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+  enable_db_subnet_group = true
 }
 
 module "db" {
-  source                     = "modules/db"
-  app-name                   = local.app-name
-  id_of_vpc                  = module.networking.id_of_vpc
+  source                     = "./modules/db"
+  app_name                   = local.app_name
+  availability_zones         = slice(data.aws_availability_zones.available.names, 0, 3)
+  database_name              = "dbname"
+  vpc_id                     = module.networking.vpc_id
   vpc_rds_security_group_ids = [module.sg.vpc_rds_security_group_ids]
-  both_db_subnets_name       = module.networking.both_db_subnets_name
-  master_password            = local.master-password
+  db_subnet_group            = module.networking.db_subnet_group
+  master_username            = "stoyandg"
```

Comment:
why not to store `master_username` in the secret as well?
it's sensitive info, like `password`

Reply:
Done.